lib/bulk/common.js

```diff
@@ -749,14 +749,19 @@ class BulkOperationBase {
 
     // Handle to the bson serializer, used to calculate running sizes
     const bson = topology.bson;
-
     // Set max byte size
     const isMaster = topology.lastIsMaster();
-    const maxBatchSizeBytes =
+
+    // If we have autoEncryption on, batch-splitting must be done on 2mb chunks, but single documents
+    // over 2mb are still allowed
+    const autoEncrypter = topology.s.options && topology.s.options.autoEncrypter;
```

Comment:
nit: you're not using the encrypter, I think this should be renamed `usingAutoEncryption` or something similar